node-mixin/dashboards/node.libsonnet

```diff
@@ -0,0 +1,176 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+local promgrafonnet = import '../lib/promgrafonnet/promgrafonnet.libsonnet';
+local gauge = promgrafonnet.gauge;
+
+{
+  grafanaDashboards+:: {
+    'nodes.json':
+      local idleCPU =
+        graphPanel.new(
+          'Idle CPU',
+          datasource='$datasource',
+          span=6,
+          format='percent',
+          max=100,
+          min=0,
+        )
+        .addTarget(prometheus.target(
+          |||
+            100 - (avg by (cpu) (irate(node_cpu{%(nodeExporterSelector)s, mode="idle", instance="$instance"}[5m])) * 100)
+          ||| % $._config,
+          legendFormat='{{cpu}}',
+          intervalFactor=10,
+        ));
+
+      local systemLoad =
+        graphPanel.new(
+          'System load',
+          datasource='$datasource',
+          span=6,
+          format='percent',
+        )
+        .addTarget(prometheus.target('node_load1{%(nodeExporterSelector)s, instance="$instance"} * 100' % $._config, legendFormat='load 1m'))
```

Comment:
Why are you treating load average as a %? I'm not sure it's a metric that we should even be displaying.

Reply:
Yeah, this should just be `format='short'`, I guess.

I'd expect a bunch of people who really like load average, and then there might be a subset of them who can actually deduct useful information from it. For now, I'll leave it in, with a TODO note.